fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportPartitionCollector.java

```diff
@@ -79,16 +78,7 @@ public Serializable collectPartitionData() throws Exception{
 
         // If the job is being stopped or in other status except for "started", then do cleanup for the partition.
         if (!batchStatus.equals(BatchStatus.STARTED)) {
-            if (partitionSummaryData.getInputStream() != null) {
-                if (partitionSummaryData.getInputStream() instanceof S3ObjectInputStream) {
-                    ((S3ObjectInputStream)partitionSummaryData.getInputStream()).abort();
-                }
-                partitionSummaryData.getInputStream().close();
-            }
-
-            if (partitionSummaryData.getBufferReader() != null) {
-                partitionSummaryData.getBufferReader().close();
-            }
+            BulkDataUtils.cleanup4TransientUserData(partitionSummaryData, true);
```

Comment:
I suggested a signature change in a prior comment. 
```suggestion
            BulkDataUtils.cleanupForTransientUserData(partitionSummaryData, true);
```

Reply:
make sense

Reply:
done